Sources/NetworkService/NetworkError/NetworkError+ErrorDescription.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+extension NetworkError: LocalizedError {
+    /// Provides a description of errors that may occur, while the application is running.
+    public var errorDescription: String? {
+        
+        switch self {
+        case .lostConnection:
+            return "lostConnection"
+        case .serverUnavailable:
+            return "serverUnavailable"
```

Comment:
Could you provide more descriptive and user-friendly message for users?
```suggestion
public var errorDescription: String? {
    switch self {
    case .lostConnection:
        return "Lost connection to the server. Please check your internet connection and try again."
    case .serverUnavailable:
        return "The server is currently unavailable. Please try again later."
    }
}
```

Reply:
I tried to improve the description.